org-formation/_scp.yaml

```diff
@@ -0,0 +1,97 @@
+  RestrictUnusedRegionsSCP:
+    Type: OC::ORG::ServiceControlPolicy
+    Properties:
+      PolicyName: RestrictUnusedRegions
+      Description: Restrict Unused regions
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Sid: DenyUnsupportedRegions
+            Effect: Deny
+            NotAction:
+              - 'cloudfront:*'
+              - 'iam:*'
+              - 'route53:*'
+              - 'support:*'
+            Resource: '*'
+            Condition:
+              StringNotEquals:
+                'aws:RequestedRegion':
+                  - us-west-1
+                  - us-west-2
+                  - us-east-1
+                  - us-east-2
+
+  RestrictDisableCloudwatchSCP:
+    Type: OC::ORG::ServiceControlPolicy
+    Properties:
+      PolicyName: RestrictDisableCloudwatch
+      Description: Restrict users from disabling cloudwatch
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Sid: DenyDisableCloudwatch
+            Effect: Deny
+            Action:
+              - 'config:DeleteConfigRule'
+              - 'config:DeleteConfigurationRecorder'
+              - 'config:DeleteDeliveryChannel'
+              - 'config:StopConfigurationRecorder'
+            Resource: '*'
+
+  RestrictDisableCloudtrailSCP:
+    Type: OC::ORG::ServiceControlPolicy
+    Properties:
+      PolicyName: RestrictDisableCloudtrail
+      Description: Restrict users from disabling cloudtrail
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Sid: DenyDisableCloudtrail
+            Effect: Deny
+            Action:
+              - 'cloudtrail:StopLogging'
+              - 'cloudtrail:DeleteTrail'
+            Resource: '*'
+
+  RestrictDisableGuarddutySCP:
```

Comment:
this is not in use yet, https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/010-scps/_tasks.yaml#L4